script/release

```diff
@@ -86,11 +86,8 @@ verbose("git tag -d #{tag} || true")
 verbose("git tag #{tag}")
 verbose("git tag #{semvertag} -m \"Release #{semvertag}.\"")
 
```

Comment:
why is there two tags?

Reply:
This is from the old publish script. We already tag twice with the old one. Richard and Andrew said we need to make sure nobody uses them before removing one of them so i leave it there.